google-cloud-clients/google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageRetryHandler.java

```diff
@@ -34,26 +30,30 @@
   private long totalWaitTime; // in milliseconds
   private final int maxRetries;
   private final int maxReopens;
+  private final CloudStorageConfiguration config;
 
   /**
    * Create a CloudStorageRetryHandler with the maximum retries and reopens set to the same value.
    *
-   * @param maxRetriesAndReopens value for both maxRetries and maxReopens
```

Comment:
This change looks like a breaking API change. Is this intentional?

We can add the new constructor, deprecate this one and then remote the deprecated one later?

Reply:
You're right, this is a public method so it's part of the API (even though I don't know of anyone using this class directly). I've added the old one back as you suggested, marked as deprecated.